www/data/showcase.json

```
@@ -1,4 +1,10 @@
 [
+  {
+    "title": "bakul plastik",
+    "link": "https://ms.chinaboxsale.com",
+    "github": "Boytobeaman/ms.chinaboxsale.com",
+    "image": "ms.chinaboxsale.com"
+  },,
```

double comma could be cleaned up
```suggestion
  },
```